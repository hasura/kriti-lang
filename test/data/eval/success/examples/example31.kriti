[
  "{{ $[1].friends[1] }}",
  "{{ $[1].friends }}",
  "{{ $?[1000] }}",
  "is: {{ $?[1000] }}"
]