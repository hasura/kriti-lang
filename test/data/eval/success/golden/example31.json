[
    "{\"id\":1,\"name\":\"Charmaine Tate\"}",
    "[{\"id\":0,\"name\":\"Shanna Emerson\"},{\"id\":1,\"name\":\"Charmaine Tate\"},{\"id\":2,\"name\":\"Elisa Albert\"}]",
    "null",
    "is: null"
]
